File: src/main/java/br/com/correios/enderecador/bean/GrupoDestinatarioBean.java
package br.com.correios.enderecador.bean;

public class GrupoDestinatarioBean {
    private String grp_nu;

    private String des_nu;

    private static GrupoDestinatarioBean grupoDestinatarioBean;

    public static GrupoDestinatarioBean getInstance() {
        if (grupoDestinatarioBean == null)
            grupoDestinatarioBean = new GrupoDestinatarioBean();
        return grupoDestinatarioBean;
    }

    public String getNumeroGrupo() {
        return this.grp_nu;
    }

    public void setNumeroGrupo(String numero) {
        this.grp_nu = numero;
    }

    public String getNumeroDestinatario() {
        return this.des_nu;
    }

    public void setNumeroDestinatario(String numero) {
        this.des_nu = numero;
    }
}

File: src/main/kotlin/br/com/correios/enderecador/bean/GrupoDestinatarioBean.kt
package br.com.correios.enderecador.bean

class GrupoDestinatarioBean {
    var numeroGrupo: String? = null
    var numeroDestinatario: String? = null

    companion object {
        private var grupoDestinatarioBean: GrupoDestinatarioBean? = null
        @JvmStatic
        val instance: GrupoDestinatarioBean?
            get() {
                if (grupoDestinatarioBean == null) {
                    grupoDestinatarioBean = GrupoDestinatarioBean()
                }
                return grupoDestinatarioBean
            }
    }
}
